assets/sass/_container.scss

```diff
@@ -0,0 +1,48 @@
+#okta-sign-in.auth-container {
+
+  // Html and body styles in _base.scss that were moved here
+  height: 100%;
+  font-family: $fonts;
+  font-size: $font-size;
+  font-weight: $font-weight-body;
+  line-height: 1.4;
+  background-color: $secondary-bg-color;
+  min-width: 100%;
```

Comment:
We should remove the properties in this section that are being overridden below.